lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart

```diff
@@ -94,80 +84,86 @@ SizedBox initialSetupWelcome(BuildContext context) {
   );
 }
 
-/// Creates a row widget displaying a piece of profile information.
-///
-/// This function constructs a `Row` widget designed to display a single piece
-/// of information in a profile UI. It is primarily used for laying out text-based
-/// information such as names, titles, or other key details in the profile section.
+// comment out the following function as it is not used in the current version
+// of the app, anushka might need to use to in the future so keeping it here.
```

Comment:
Hi @anushkavidanage , could you please have a look at the following part, if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)